types/sharedb/sharedb-tests.ts

```diff
@@ -43,6 +43,9 @@ const backend = new ShareDB({
     milestoneDb: new MyMilestoneDB(),
     suppressPublish: false,
     maxSubmitRetries: 3,
+    errorHandler: (error) => {
```

Comment:
❗  The 2-arg handler should be exercised here, so that the test will fail if there's a breaking change to the error context.